### DISCUSSION TITLE
4.0.0

### DISCUSSION THREAD

**Discussion by nalexandru** [nalexandru/BlenderCompat#70] [Announcements]
Blender 4.0.0, commit [878f71061b8e](https://projects.blender.org/blender/blender/commit/878f71061b8e).
Build environment:

    Visual Studio 2019, version 16.11.31
    Windows SDK 10.0.22621
    NVIDIA CUDA 11.8
    NVIDIA OptiX 7.3

AMD HIP and SM3.0 kernel binaries copied from the official release.

The required OpenGL version is now 4.3 and Intel HD4000 support was dropped.
Please read the Graphics Cards section of the release notes: [https://wiki.blender.org/wiki/Reference/Release_Notes/4.0](https://wiki.blender.org/wiki/Reference/Release_Notes/4.0)

<hr /><em>This discussion was created from the release <a href='https://github.com/nalexandru/BlenderCompat/releases/tag/4.0.0'>4.0.0</a>.</em>

**Comment by atekdigital** [comment]
Why Blender not starting. I have download the blender-4.0.0-windows-7 file and unzip and open it. After clicking on blender exe it open, Then i click on previous version setting to update. after updating previous version setting, blender closed, and then try to open blender does not. I try to restart my pc 4-5 times and tried but did not work. Any solution or help why blender unable to start. Thanks

**Reply by nalexandru** [reply]
You can reset the settings by moving or deleting %APPDATA%\Blender Foundation\Blender\4.0. After that, make your settings manually and see what breaks it.

**Reply by atekdigital** [reply]
deleting %APPDATA%\Roaming\Blender Foundation\Blender\4.0 folder did not work, still same blender does not start after deleting the folder also. Any other solution, please let me know. Thx

**Comment by atekdigital** [comment]
deleting %APPDATA%\Roaming\Blender Foundation\Blender\4.0 folder did not
work, still same blender does not start after deleting the folder also. Any
other solution, please let me know. Thx

On Sat, 18 Nov 2023 at 22:15, Alexandru Naiman ***@***.***>
wrote:

> You can reset the settings by moving or deleting %APPDATA%\Roaming\Blender
> Foundation\Blender\4.0. After that, make your settings manually and see
> what breaks it.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nalexandru/BlenderCompat/discussions/70#discussioncomment-7607321>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFQ5F2AXEHPQRAYA4H4JBI3YFDQ2DAVCNFSM6AAAAAA7LIYF32VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TMMBXGMZDC>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 
Atek Digital
